Specialization.cs

```diff
@@ -57,6 +57,19 @@ public static string GetCoreName(uint core)
             };
         }
 
+        public static readonly IReadOnlyCollection<uint> ProfessionCodes = new uint[]
```

Comment:
Nitpick: I think it should be similar to the initialization below (`public static readonly IReadOnlyCollection<int> EliteCodes = new[]`), so either use `new[]` here, or expand to `new uint[]` below!